app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[edit update destroy]
+
+  # GET /users
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1/edit
+  def edit; end
+
+  # PATCH/PUT /users/1
+  def update
+    if @user.update(user_params)
+      format.html { redirect_to users_url, notice: "User was successfully updated." }
```

Comment:
This shouldn't be working. `format` doesn't exists, it was part of the `respond_to`
Let's get rid of them

Reply:
format.html has been removed and the routes are working fine